misc/syntax-api-calls-gen/src/main/resources/api_gen_syntax_tree_descriptor.json

```diff
@@ -4211,7 +4211,7 @@
         },
         {
           "name": "variableName",
```

Comment:
We can call this `simpleBindingPattern` as `variableName` could be misleading? @rdulmina wdyt?
(In the AST we have `BLangSimpleBindingPattern` to represent both capture and wildcard binding patterns.)

Reply:
+1

Reply:
@suleka96 please note.